dpnp/dpnp_iface.py

```diff
@@ -516,3 +517,46 @@ def is_supported_array_type(a):
     """
 
     return isinstance(a, (dpnp_array, dpt.usm_ndarray))
+
+
+def _copyto(a, out=None):
+    """
+    If `out` is provided, `a` will be inserted into this array.
+    Otherwise, `a` is returned.
```

Comment:
```suggestion
    If `out` is provided, values of `a` array will be copied into the out array according to `safe` casting rule.
    Otherwise, the input array `a` is returned.
```